tools/dos/web/HashCollision-DOS-POC/HashtablePOC.py

```diff
@@ -314,7 +325,7 @@ def main():
 
         if options.verbose:
             if len(request) > 400:
-                print(request[:400]+"....")
+                print(request[:400] + "....")
```

Comment:
String concat using `%` and No space around operators in args for the rest of the file. (below)